fe/fe-core/src/main/java/org/apache/doris/transaction/PublishVersionDaemon.java

```diff
@@ -63,13 +58,33 @@ protected void runAfterCatalogReady() {
         }
     }
 
-    private boolean isAllBackendsOfUnfinishedTasksDead(List<PublishVersionTask> unfinishedTasks) {
-        for (PublishVersionTask unfinishedTask : unfinishedTasks) {
-            if (Env.getCurrentSystemInfo().checkBackendAlive(unfinishedTask.getBackendId())) {
-                return false;
+    private void genPublishErrorReplicas(TransactionState transactionState) {
+        TabletInvertedIndex tabletInvertedIndex = Env.getCurrentInvertedIndex();
+        Map<Long, Set<Long>> publishErrorReplicas = Maps.newHashMap();
+        for (PublishVersionTask task : transactionState.getPublishVersionTasks().values()) {
+            long backendId = task.getBackendId();
+            if (!task.isFinished()) {
+                continue;
+            }
+            Set<Long> errorReplicas = Sets.newHashSet();
+            for (Long tabletId : task.getErrorTablets()) {
+                if (tabletInvertedIndex.getTabletMeta(tabletId) == null) {
+                    continue;
```

Comment:
We should print a log here, it means a balance or drop operation happens.

Reply:
done print log